src/cloudWatchLogs/registry/logStreamRegistry.ts

```diff
@@ -85,6 +86,11 @@ export class LogStreamRegistry {
                 // log entries containing newlines are indented to the same length as the timestamp.
                 line = line.replace(inlineNewLineRegex, `\n${timestampSpaceEquivalent}\t`)
             }
+            //Linting error here, not sure how to fix.
```

Comment:
What was the error?

Reply:
I was struggling with the `CloudWatchEvents` type from the sdk not having the stream name as a property. I was able to fix it by creating the extended `CloudWatchEvents` type. This comment is leftover from when I was stuck on it.